index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+ <meta charset="UTF-8">
+ <meta http-equiv="X-UA-Compatible" content="IE=edge">
+ <meta name="viewport" content="width=device-width, initial-scale=1.0">
+ <link rel="stylesheet" href="style.css">
+ <title>Portfolio</title>
+</head>
+<body>
```

Comment:
- [ ] Kindly add the given margin, to all of the elements of your project. These are part of the requirements.
- [ ] Use the provided font size for all the elements. As you can see your heading is bigger than the one in the provided design.

Your design
![image](https://user-images.githubusercontent.com/81932478/176912183-8dea1f6e-6d93-499a-94ff-601c92022ad5.png)

Provided design
![image](https://user-images.githubusercontent.com/81932478/176913063-11b82e4c-bdfc-45fd-86e3-7ba44db6e182.png)


- [ ] Notice how there is a difference between your design and the provided design. the Icons at the bottom are not properly aligned.